changelogs/fragments/68550-ansible-test-docs-changelogs.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+- "ansible-test change detection - ignore ``docs/`` and ``changelogs/`` in collections, to avoid triggering full CI runs when files in these directories change."
```

Comment:
```suggestion
- "ansible-test change detection - Run only sanity tests on ``docs/`` and ``changelogs/`` in collections, to avoid triggering full CI runs of integration and unit tests when files in these directories change."
```